target/linux/ar71xx/files/arch/mips/ath79/mach-ap531b0.c

```diff
@@ -0,0 +1,121 @@
+
+/*
+ * Rockeetech AP531B0 11ng wireless AP board support
+ *
+ *  Copyright (C) 2013 Gabor Juhos <juhosg@openwrt.org>
+ *  Copyright (C) 2016 roboidler <roboidler@gmail.com>
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License version 2 as published
+ *  by the Free Software Foundation.
+ */
+
+#include <linux/platform_device.h>
+#include <linux/ath9k_platform.h>
+#include <linux/ar8216_platform.h>
+
+#include <asm/mach-ath79/ar71xx_regs.h>
+
+#include "common.h"
+#include "dev-eth.h"
+#include "dev-gpio-buttons.h"
+#include "dev-leds-gpio.h"
+#include "dev-m25p80.h"
+#include "dev-spi.h"
+#include "dev-usb.h"
+#include "dev-wmac.h"
+#include "machtypes.h"
+#include "pci.h"
+#include "dev-ap9x-pci.h"
+
+
+
+#define AP531B0_GPIO_LED_12		12
+#define AP531B0_GPIO_LED_11		11
+
+
+#define AP531B0_GPIO_RST_BTN		17
+
+#define AP531B0_KEYS_POLL_INTERVAL	20	/* msecs */
+#define AP531B0_KEYS_DEBOUNCE_INTERVAL	(3 * AP531B0_KEYS_POLL_INTERVAL)
+
+#define AP531B0_MAC0_OFFSET		0
+#define AP531B0_MAC1_OFFSET		6
+#define AP531B0_WMAC_CALDATA_OFFSET	0x1000
+
+
+static struct gpio_led ap531b0_leds_gpio[] __initdata = {
+	{
+		.name		= "ap531b0:green:g11",
```

Comment:
Dear pepe2k,

this device is just a PCBA without casing,  which has two gpio( gpio 11 and gpio 12) controlled led, fully user defined